vertx-pojo-mapper-common/pom.xml

```diff
@@ -25,4 +25,11 @@
 	<description>common interfaces and implementations for pojo mapper
 	</description>
 
+	<dependencies>
+		<dependency>
+			<groupId>commons-codec</groupId>
+			<artifactId>commons-codec</artifactId>
+			<version>1.4</version>
```

Comment:
vertx-util already has this dependency in its gradle file, and with version 1.10